static/images/mastodon.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" shape-rendering="geometricPrecision" text-rendering="geometricPrecision" image-rendering="optimizeQuality" fill-rule="evenodd" clip-rule="evenodd" viewBox="0 0 480 511.475"><path fill="#fff" fill-rule="nonzero" d="M478.065 113.237c-7.393-54.954-55.29-98.266-112.072-106.656C356.413 5.162 320.121 0 236.043 0h-.626c-84.099 0-102.141 5.162-111.72 6.581C68.498 14.738 18.089 53.653 5.859 109.26c-5.882 27.384-6.51 57.747-5.416 85.594C2 234.794 2.302 274.662 5.93 314.438a562.647 562.647 0 0013.09 78.437c11.625 47.653 58.686 87.311 104.793 103.494a281.07 281.07 0 00153.315 8.09 223.879 223.879 0 0016.578-4.533c12.368-3.929 26.855-8.323 37.507-16.043a1.198 1.198 0 00.486-.928v-38.544a1.12 1.12 0 00-.437-.879 1.134 1.134 0 00-.957-.215 424.764 424.764 0 01-99.49 11.626c-57.665 0-73.172-27.361-77.611-38.75a120.008 120.008 0 01-6.745-30.549 1.048 1.048 0 01.094-.51c.07-.161.178-.304.317-.415a1.094 1.094 0 01.982-.213 417.004 417.004 0 0097.865 11.624c7.927 0 15.834 0 23.761-.213 33.156-.928 68.105-2.625 100.722-8.997.816-.16 1.63-.301 2.327-.508 51.455-9.883 100.423-40.894 105.397-119.421.184-3.092.652-32.382.652-35.588.021-10.904 3.509-77.344-.511-118.166zm-82.796 61.744V311.51h-54.104V179.003c0-27.897-11.627-42.126-35.273-42.126-25.998 0-39.018 16.834-39.018 50.073v72.534h-53.776V186.95c0-33.239-13.045-50.073-39.04-50.073-23.507 0-35.249 14.229-35.249 42.126V311.51H84.73V174.981c0-27.896 7.121-50.059 21.366-66.488 14.693-16.388 33.969-24.805 57.893-24.805 27.692 0 48.618 10.648 62.569 31.919l13.464 22.597 13.485-22.597c13.95-21.271 34.876-31.919 62.52-31.919 23.903 0 43.177 8.417 57.919 24.805 14.231 16.413 21.337 38.576 21.323 66.488z"/></svg>
```

Comment:
we might want to normalize on using square SVGs for these icons, but some others are also not squares and it seems like this works well enough for now 👍